dll/win32/ws2help/ws2help.spec

```diff
@@ -1,23 +1,47 @@
-@ stdcall WahCloseApcHelper(long)
-@ stdcall WahCloseHandleHelper(long)
-@ stdcall WahCloseNotificationHandleHelper(long)
-@ stdcall WahCloseSocketHandle(long ptr)
-@ stdcall WahCloseThread(long ptr)
-@ stdcall WahCompleteRequest(long ptr ptr long long)
-@ stdcall WahCreateHandleContextTable(ptr)
-@ stdcall WahCreateNotificationHandle(long ptr)
-@ stdcall WahCreateSocketHandle(long ptr)
-@ stdcall WahDestroyHandleContextTable(ptr)
-@ stdcall WahDisableNonIFSHandleSupport()
-@ stdcall WahEnableNonIFSHandleSupport()
-@ stdcall WahEnumerateHandleContexts(ptr ptr ptr)
-@ stdcall WahInsertHandleContext(ptr ptr)
-@ stdcall WahNotifyAllProcesses(long)
-@ stdcall WahOpenApcHelper(ptr)
-@ stdcall WahOpenCurrentThread(long ptr)
-@ stdcall WahOpenHandleHelper(ptr)
-@ stdcall WahOpenNotificationHandleHelper(ptr)
-@ stdcall WahQueueUserApc(long ptr ptr ptr)
-@ stdcall WahReferenceContextByHandle(ptr long)
-@ stdcall WahRemoveHandleContext(ptr ptr)
-@ stdcall WahWaitForNotification(long long ptr ptr)
+@ stdcall -version=0x400-0x503 WahCloseApcHelper(long)
```

Comment:
Do we need a lower-range specification ("0x400") for these, or is it possible to specify something like `-version=-0x503` ?

Reply:
no, that doesn't work